common/src/main/java/de/maxhenkel/voicechat/config/ClientConfig.java

```diff
@@ -163,48 +163,48 @@ public ClientConfig(ConfigBuilder builder) {
                 );
         denoiser = builder
                 .booleanEntry("denoiser", false,
-                        "If noise cancellation should be enabled"
+                        "Specifies whether noise suppression should be enabled"
                 );
         runLocalServer = builder
                 .booleanEntry("run_local_server", true,
-                        "If voice chat should work in singleplayer/LAN worlds"
+                        "Controls whether the voice chat should work in singleplayer or in worlds shared over LAN"
                 );
         javaMicrophoneImplementation = builder
                 .booleanEntry("java_microphone_implementation", Platform.isMac(),
-                        "Whether to use the Java implementation of microphone capturing instead of OpenAL"
+                        "Controls whether to use the Java implementation of microphone capture instead of OpenAL"
                 );
         macosCheckMicrophonePermission = builder
                 .booleanEntry("macos_check_microphone_permission", true,
-                        "If the mod should check for microphone permissions (MacOS only)"
+                        "Controls whether the mod should check for microphone permissions (macOS only)"
                 );
         showFakePlayersDisconnected = builder
                 .booleanEntry("show_fake_players_disconnected", false,
-                        "If fake players should have the disconnected icon above their head"
+                        "Specifies whether fake players should have the disconnected icon above their head"
                 );
         offlinePlayerVolumeAdjustment = builder
                 .booleanEntry("offline_player_volume_adjustment", false,
-                        "If the volume adjustment GUI should also show offline players"
+                        "Specifies whether the volume adjustment interface should also display offline players"
                 );
         audioType = builder
                 .enumEntry("audio_type", AudioType.NORMAL,
                         "The 3D audio type",
-                        "Possible values are 'NORMAL', 'REDUCED' and 'OFF'"
+                        "Valid values are 'NORMAL', 'REDUCED', and 'OFF'"
                 );
         useNatives = builder
                 .booleanEntry("use_natives", true,
-                        "If the mod should load native libraries",
-                        "If set to false, the Java Opus implementation will be used, the denoiser won't be available and you won't be able to record audio."
+                        "Controls whether the mod should load native libraries",
```

Comment:
Why change this? Now you used "Controls whether" again. Again, no need to change this.